modules/develop/pages/http-proxy-cloud.adoc

```diff
@@ -0,0 +1,6 @@
+= Use Redpanda with the HTTP Proxy API
```

Comment:
Yes, this will live in cloud-docs, but right now it's in docs. I thought we move stuff like this after we merge cloud-docs, no? Can you please remind me of the naming convention we should use? We have http-proxy.adoc + now a partial named http-proxy.adoc and http-proxy-cloud.adoc. What process/naming do you recommend?

Reply:
if the file is being moved to cloud-docs, I'd say we don't need the prefix.

Reply:
@micheleRP, There are 2 files with the same content http-proxy.adoc and http-proxy-cloud.adoc. Is that correct?